contrib/cmd/user-broker/user-broker.go

```diff
@@ -37,10 +42,39 @@ func init() {
 }
 
 func main() {
+	if err := run(); err != nil && err != context.Canceled && err != context.DeadlineExceeded {
+		glog.Fatalln(err)
```

Comment:
@nilebox nit, not blocking this PR: can you still log on cancelled or timed-out contexts?